content/bookings/intro.en.md

```diff
@@ -1,6 +1,5 @@
 # **Welcome!**
 
-For some of our activities, we kindly ask you to make a reservation.
-On this page, you can book a spot at one of our events.
+To join our Monday dinner, we kindly ask you to make a reservation.
 We will confirm your reservation by email.
-In case you're not able to join the event, please cancel the reservation through the cancelation link in the confirmation email.
+In case you're not able to join the event, please cancel the reservation through the cancellation link in the confirmation email.
```

Comment:
```suggestion
In case you're not able to join the event, please cancel the reservation through the cancelation link in the confirmation email.
```
we attempt to use american english